gazebo_msgs/srv/AddForceSensor.srv

```diff
@@ -0,0 +1,3 @@
+string joint_name
```

Comment:
Add comment to explain where the sensor is added for the joint? Same for the torque sensor service